x-pack/plugins/security_solution/common/endpoint/data_loaders/index_endpoint_fleet_actions.ts

```diff
@@ -47,7 +47,7 @@ export const indexEndpointAndFleetActionsForHost = async (
 ): Promise<IndexedEndpointAndFleetActionsForHostResponse> => {
   const ES_INDEX_OPTIONS = { headers: { 'X-elastic-product-origin': 'fleet' } };
   const agentId = endpointHost.elastic.agent.id;
-  const total = fleetActionGenerator.randomN(5) + 1; // generate at least one
+  const total = fleetActionGenerator.randomN(5) + 11; // generate at least 11 actions
```

Comment:
Do we really want 11 actions to be index every time the generator runs? Remember that this is used in other use cases outside of your test. 

Suggestion: consider adding an `options` argument to this function and introduce a `count` parameter that drives how many to create and then use that from your test.

Reply:
Yeah, adding options is a good idea.

Reply:
ca1919b043ff5715dadc5245e36d5e3f284e8c1e